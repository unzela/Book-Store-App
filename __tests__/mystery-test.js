import 'react-native';
import React from 'react';
import mysteryBooks from '../src/data/mystery';
import renderer from 'react-test-renderer';

test('Mystery Snapshot',()=>{
    const snap = renderer.create(
        <mysteryBooks/>
    ).toJSON();

    expect(snap).toMatchSnapshot();
});

test('Mystery Renderer',()=>{
    renderer.create(
        <mysteryBooks/>
    );

  
});
