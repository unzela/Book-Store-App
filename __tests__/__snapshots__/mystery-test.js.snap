// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mystery Snapshot 1`] = `<mysteryBooks />`;
